schema.graphql

```diff
@@ -219,8 +219,12 @@ type GraphNetwork @entity {
   slashingPercentage: Int!
   "Minimum deposit to create a dispute"
   minimumDisputeDeposit: BigInt!
-  "Reward to Fisherman on successful disputes. In parts per million"
+  "[LEGACY] Reward to Fisherman on successful disputes. In parts per million"
   fishermanRewardPercentage: Int!
+  "[HORIZON]Reward to Fisherman on successful disputes. In parts per million"
+  fishermanRewardCut: Int!
+  "Maximum Penalty to Indexer on successful disputes for indexing disputes. In parts per million"
```

Comment:
Might be worth saying this is a percentage of the indexer's stake snapshot (own stake plus delegated).